auth/http.go

```diff
@@ -1266,6 +1263,22 @@ func httpHandlerSetGroupRoles(central *ImqsCentral, w http.ResponseWriter, r *ht
 	broadcastGroupChange(central, r, groupname)
 }
 
+func permListToText(central *ImqsCentral, added *authaus.PermissionList) string {
```

Comment:
```suggestion
func permListToText(central *ImqsCentral, permList *authaus.PermissionList) string {
```
I would rather call the 2nd parameter `permList` as `added` in the context of the function sounds like you are appending to something.
Since the function is called **permList**ToText, the parameter named permList sounds more appropriate.